### ISSUE TITLE
Usage in New Architecture projects

### ISSUE THREAD

**Issue by jay-jlm** [callstack/react-native-brownfield#76]
We have an upcoming project that is expected to use RN 73 and we'd like if this package could still be relevant for us.

**Comment by okwasniewski** [comment]
Hey, new architecture support is planned to land soon. 

There is an open PR for iOS: #79

**Comment by yzhe554** [comment]
@okwasniewski  any update on the PR? Will android also be supported?

**Comment by okwasniewski** [comment]
Yeah, I'm planning also to support Android but I'm working on this in my free time so it can take some time. I'll try to merge iOS PR this week

**Comment by pawel3ala** [comment]
hey @okwasniewski how things are going on the Android side?

**Comment by yzhe554** [comment]
Try to follow up what's the expectation for this issue. @okwasniewski

**Comment by okwasniewski** [comment]
@yzhe554 It's supported in latest release, give it a try 😄
